echopype/visualize/api.py

```diff
@@ -0,0 +1,123 @@
+from typing import Union, List
+
+import xarray as xr
+
+from echopype.visualize.plot import _plot_echogram, FacetGrid, QuadMesh, T
+from echopype.echodata import EchoData
+
+
+def create_echogram(
+    data: Union[EchoData, xr.Dataset],
+    frequency: Union[int, float, List[T], None] = None,
+    get_range: bool = False,
+    range_kwargs: dict = {},
+    water_level: Union[xr.DataArray, None] = None,
+    **kwargs,
+) -> Union[FacetGrid, QuadMesh]:
+    """Create an Echogram from an EchoData object or Sv and MVBS Dataset.
+
+    Parameters
+    ----------
+    data : EchoData or xr.Dataset
+        Echodata or Xarray Dataset to be plotted
+    frequency : int, float, or list of float or ints, optional
+        The frequency to be plotted.
+        Otherwise all frequency will be plotted.
+    get_range : bool
+        Flag as to whether range should be computed or not,
+        by default it will just plot range_bin as the yaxis.
+    range_kwargs : dict
+        Keyword arguments dictionary for computing range.
+        Keys are `env_params`, `waveform_mode`, and `encode_mode`.
+    water_level : xr.DataArray, optional
+        Water level data array for platform water level correction.
+    **kwargs: optional
+        Additional keyword arguments for xarray plot pcolormesh.
+
+    """
+    range_attrs = {
+        'long_name': 'Range',
+        'units': 'm',
+    }
```

Comment:
Does CF convention has a long_name for range? It may be worth checking since the same wording/concept is applicable to radar data also.

Reply:
I don't see any "range" CF `standard_name` entry that are relevant for sonars. See http://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html and type "range" in the text box.

Reply:
Yea the only "range" i see is `altimeter_range` which doesn't really makes sense in this case.